### DISCUSSION TITLE
DAT 2.0

### DISCUSSION THREAD

**Discussion by mlrx-potindji** [dat-linux/community#73] [General]
Could it be possible to have a support on how to upgrade our current 1.0.2.1 to the newest 2. version ???

Thanks

**Comment by dat-linux** [comment]
Hi. This will be posted shortly. ✔️

**Comment by dat-linux** [comment]
Please see: https://github.com/dat-linux/community/discussions/74 (document in progress)

**Comment by mlrx-potindji** [comment]
Thanks, i just completed the upgrade, now on the 2.0 stream
